aten/src/ATen/Context.h

```diff
@@ -37,10 +38,14 @@ class AT_API Context {
 
     return type;
   }
-  Type & getType(Backend p, ScalarType s) {
+  Type & getType(Backend p, ScalarType s, bool is_variable = false) {
```

Comment:
not: I generally dislike default args when it's not totally obvious what the default parameter should be, because it's easy to make a mistake without even realizing you were making a mistake (by e.g. copy pasting).  Are there a lot of call sites here or can we just update them and get rid of default?

Reply:
There is a non-trivial number, but not so many that the refactor would be unmanageable. Should be simple enough to fix.